tests/bluetooth/audio/cap_commander/src/test_volume.c

```diff
@@ -0,0 +1,552 @@
+/* test_change_volume.c - unit test for volume settings */
```

Comment:
Mismatch between file name and comment

Reply:
Fixed the comment, but maybe we shouldn't have the filename in the comment at all